src/languages/postgresql/postgresql.formatter.ts

```diff
@@ -23,6 +23,7 @@ const reservedClauses = expandPhrases([
   // - insert:
   'INSERT INTO',
   'VALUES',
+  'DEFAULT VALUES',
```

Comment:
I wasn't sure exactly which of the "clauses" arrays this belongs in, though it seems to not matter for the tests. If you'd rather this be moved to another array, please let me know.